javascript/pigeonholes.js

```diff
@@ -0,0 +1,183 @@
+var rowValues = [2, 3, 4]

```

Comment:
Can you add a quick comment next to each of these variables explaining what these stand for